build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'java'
+    id 'checkstyle'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.itextpdf:itextpdf:5.5.13.1'
+    implementation 'org.bouncycastle:bcpkix-jdk15on:1.63'
+    implementation 'org.jetbrains:annotations:16.0.2'
+    implementation 'javax.xml.bind:jaxb-api:2.2.4'
+
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.3.1'
+
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
+    testImplementation 'org.mockito:mockito-core:3.0.0'
+    testImplementation 'org.mockito:mockito-junit-jupiter:2.23.0'
+}
+
+test {
+    useJUnitPlatform()
+    maxHeapSize = '1G'
+    testLogging {
+        outputs.upToDateWhen {false}
+        showStandardStreams = true
+    }
+}
+
+jar {
+  manifest {
+    attributes(
+      'Main-Class': 'com.advanon.pdfsignatures.Application'
+    )
+  }
+  from (configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }) {
+    exclude 'META-INF/MANIFEST.MF'
+    exclude 'META-INF/*.SF'
+    exclude 'META-INF/*.DSA'
+    exclude 'META-INF/*.RSA'
+  }
+}
+
+checkstyle {
+  toolVersion "8.6"
+  sourceSets = [project.sourceSets.main]
+}
+
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'
+
+version = '0.0.1'
+
+task copyJarToLibs(type: Copy) {
+  from 'build/libs/pdf-signatures-' + version + '.jar'
```

Comment:
Do we need to check the binary file `libs/pdf-signatures-0.0.1.jar` (7.5MB) in to the repo, or could we create it as part of a build task that packages the module?

Reply:
yep, that was also suggested by Andrea, will make it part of the pipeline